content/conscious-coliving.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Conscious Coliving: A Way of Living to Thrive"
+description: Living together in conscious communities is a concrete practical action to enable personal, collective and societal flourishing.
+---
+![[conscious-coliving-page-header.jpg]]
+
+### Living together in conscious communities is a concrete practical action to enable personal, collective and societal flourishing.
+
+[[conscious-coliving-course|New to conscious coliving? Start here.]]
+
+## What is Conscious Coliving? 
+
+There’s not really an established term yet for the kind of way of living that we imagine and seek to pioneer. We've come to call it "conscious coliving" reflecting the fusion of two related practices: conscious living and coliving.
+
+It is the practice of living in community, in any format, and dedicating oneself to conscious living and embodiment. The spectrum of conscious coliving environments is broad and can be anything from three friends living in a house together, to an entire village. 
+
+[[what-is-conscious-coliving|For a more detailed definition read our article: What is conscious coliving?]]
+
+## Learn About Conscious Coliving
+
+If you are curious about conscious coliving and want to implement it into your life, our **free email courses** are a great place to begin. 
+
+### Out Now
+
+[[conscious-coliving-course|Conscious Coliving 101:Learn How to Live a More Connected Life]]
+
+### Coming Soon (Pre-Apply Now!)
+
+
+
+### The Online Community for Conscious Coliving
+
+Join 100+ other curious conscious colivers to learn, share and connect in a simple, easygoing way. No thrills, just a chat channel! 
+
+[Join Now](https://chat.whatsapp.com/DycZceubrTzI1DHKHZr5sw)
```

Comment:
Make this clearer. Also move this nearer the top i think as a major option.

Reply:
I've linked to the canva board, in which I made this a yellow banner. I think making it as a banner would be super useful, see: https://www.canva.com/design/DAF0NW4nRlQ/GuClM2j0-9ZLvHKkzF_wUA/view?utm_content=DAF0NW4nRlQ&utm_campaign=designshare&utm_medium=link&utm_source=editor